Marlin/language_nl.h

```diff
@@ -161,5 +166,58 @@
 #define MSG_DELTA_CALIBRATE_Y               "Kalibreer Y"
 #define MSG_DELTA_CALIBRATE_Z               "Kalibreer Z"
 #define MSG_DELTA_CALIBRATE_CENTER          "Kalibreer Midden"
-
+#define MSG_DAC_PERCENT                     "Driver %"
+#define MSG_DAC_EEPROM_WRITE                "DAC EEPROM Write"
+#define MSG_DRIVE_STRENGTH                  "Drive Strength"
+#define MSG_ERR_Z_HOMING                    "Fout Z homing"
+#define MSG_FILAMENT_CHANGE_OPTION_EXTRUDE  "Extrude meer"
+#define MSG_FILAMENT_CHANGE_HEADER          "WISSEL FILAMENT"
+#define MSG_FILAMENT_CHANGE_OPTION_RESUME   "Hervat print"
+// Up to 3 lines allowed
+#define MSG_FILAMENT_CHANGE_INIT_1          "Wacht voor start"
+#define MSG_FILAMENT_CHANGE_INIT_2          "filament te"
+#define MSG_FILAMENT_CHANGE_INIT_3          "verwisselen"
+#define MSG_FILAMENT_CHANGE_UNLOAD_1        "Wacht voor"
+#define MSG_FILAMENT_CHANGE_UNLOAD_2        "filament uit"
+#define MSG_FILAMENT_CHANGE_UNLOAD_3        "te laden"
+#define MSG_FILAMENT_CHANGE_INSERT_1        "Laad filament"
+#define MSG_FILAMENT_CHANGE_INSERT_2        "en druk knop"
+#define MSG_FILAMENT_CHANGE_INSERT_3        "om verder..."
+#define MSG_FILAMENT_CHANGE_LOAD_1          "Wacht voor"
+#define MSG_FILAMENT_CHANGE_LOAD_2          "filament te"
+#define MSG_FILAMENT_CHANGE_LOAD_3          "laden"
+#define MSG_FILAMENT_CHANGE_EXTRUDE_1       "Wacht voor"
+#define MSG_FILAMENT_CHANGE_EXTRUDE_2       "filament te"
+#define MSG_FILAMENT_CHANGE_EXTRUDE_2       "extruderen"
```

Comment:
Thanks @thinkyhead will look into that and fix the overlooked MSG_FILAMENT_CHANGE_EXTRUDE_2 line.